mathics_django/web/media/js/inout.js

```diff
@@ -259,102 +259,99 @@ function loadLink() {
+		'D[Sin[2x] + Log[x] ^ 2, x]',
+		'Integrate[Tan[x] ^ 5, x]',
+
+		'(**** Linear Algebra ****)',
+		'MagicSquare = {{2, 7, 6}, {9, 5, 1}, {4, 3, 8}}; MatrixForm[MagicSquare]',
+		'LinearSolve[MagicSquare, {1, 1, 1}] // MatrixForm',
+		'Eigenvalues[MagicSquare]',
+		'(**** Chemical Data ***)',
+		'(* 2nd and 3rd Row of Periodic Table *)',
+		'Grid[{Table[ElementData[i], {i, 3, 10}], Table[ElementData[i], {i, 11, 18}]}]',
+		'ListLinePlot[Table[ElementData[z, "MeltingPoint"], {z, 118}]]',
+
+		'(**** Some graphs ****)',
+		'ListPlot[{Sqrt[Range[40]], Log[Range[40, 80]]}, TicksStyle->{Blue,Purple}]',
+		'Plot[{Sin[x], Cos[x], Tan[x]}, {x, -3Pi, 3Pi}]',
+		'Graphics[Polygon[{{150,0},{121,90},{198,35},{102,35},{179,90}}]]',
+
+		'(* Charts: *)',
+		'BarChart[{{1, 2, 3}, {2, 3, 4}}]',
+		'PieChart[{30, 20, 10}, ChartLabels -> {Dogs, Cats, Fish}]',
+
+		'(* Bouncing Ping-Pong Ball at equal time intervals *)',
+		'Plot[Abs[Sin[t] / (t + 1)], {t, 0, 4 Pi}, Mesh->Full, PlotRange->{0, 1 / 2}]',
+
+		'(* Here is a 5-blade propeller, or maybe a flower, using PolarPlot: *)',
+		'PolarPlot[Cos[5t], {t, 0, Pi}]',
+		'(* Also try surrounding the Cos in an Abs: PolarPlot[Abs[Cos[5t]], {t, 0, Pi}] *)',
+
+		'Graphics[Table[Circle[{x,y}], {x, 0, 10, 2}, {y, 0, 10, 2}]]',
+
+		'(* Target Practice. *)',
+		'Graphics[Circle[], Axes-> True]',
+
+		'(* Random dots in Sierpinski Triangle. *)',
+		'vertices = {{0,0}, {1,0}, {.5, .5 Sqrt[3]}};',
+		'points = NestList[.5(vertices[[ RandomInteger[{1,3}] ]] + #) &, {0.,0.}, 600];',
+		'Graphics[Point[points], ImageSize->Small]',
+
+		'Graphics[Table[{EdgeForm[{GrayLevel[0, 0.5]}], Hue[(-11+q+10r)/72, 1, 1, 0.6], Disk[(8-r){Cos[2Pi q/12], Sin [2Pi q/12]}, (8-r)/3]}, {r, 6}, {q, 12}]]',
+
+		'(* Embedding objects in a Table. *)',
+		'Table[RGBColor[1, g, 0], {g, 0, 1, 0.05}]',
+		'Table[Plot[x^i Sin[j Pi x], {x, 0, 2}],{i, 2},{j, 2}]//MatrixForm',
+
+		'(**** 3D graphics ****)',
+
+		'Graphics3D[Arrow[{{1, 1, -1}, {2, 2, 0}, {3, 3, -1}, {4, 4, 0}}]]',
+		'Graphics3D[{Darker[Yellow], Sphere[{{-1, 0, 0}, {1, 0, 0}, {0, 0, Sqrt[3.]}}, 1]}]',
+		'Graphics3D[{ Cylinder[{{1,1,1}, {10,10,10}}], Cylinder[{{-1,-1,-1}, {-10,-10,-10}}] }]',
+		'(* A 3D grid of points.\n\
 Orient so the long diagonal becomes a point to see the kinds of difficulties visualizing a 3D lattice in 2D: *)\n\
 Graphics3D[Table[Point[{x, y, z}], {x, 5}, {y, 5}, {z, 5}]]',
-	    'Plot3D[Sin[x y], {x, -2, 2}, {y, -2, 2}, Mesh->Full, PlotPoints->21, TicksStyle->{Darker[Magenta], Darker[Blue]}]',
-	    'Plot3D[ Abs[Zeta[x + I y] ], {x, -1, 2}, {y, 2, 20}, PlotPoints->30]',
-
-	    '(**** Combinatorica: for Implementing Discrete Mathematics. ****)',
-	    'Needs["DiscreteMath`CombinatoricaV0.9`"]',
-	    'ShowGraph[K[6,6,6]]',
-	    'ShowGraph[ CirculantGraph[20, RandomSubset[Range[10]]] ]',
-	    'FerrersDiagram[RandomInteger[{0, 3}, 50]]'
+		'Plot3D[Sin[x y], {x, -2, 2}, {y, -2, 2}, Mesh->Full, PlotPoints->21, TicksStyle->{Darker[Magenta], Darker[Blue]}]',
```

Comment:
Hmm. It would be great to move the Image stuff  up earlier, _after_ 2D Graphics And the examples were changed slightly as well. 

Plot3D is slow which is why I think it should go towards the bottom after the images.  That way, people will be distracted by. 

https://github.com/Mathics3/mathics-django/commit/07380caa4c07e13743ce504e963bd50416b4aac6 has the details.

Reply:
Ok. I just changed this file to fix the indentation. (I'm neutral in that)

Reply:
Ok. I will try moving things around and making the small edits again. I don't know if my editor will reformat, I hope not.